src/Raven.Server/Commercial/LetsEncrypt/JsonStringHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+
+namespace Raven.Server.Commercial.LetsEncrypt;
+
+public class JsonStringHelper
+{
+    public static string IndentJsonString(string json)
```

Comment:
Just Indent? So instead of `JsonStringHelper.IndentJsonString()` you'd have `JsonStringHelper.Indent()` - it's easier to read it.

Reply:
Ok